MetaMask/metamask-mobile
# fix(2804): Right margin was too wide and looked broken compared to left

## **Description**
This fixes the right margin of the modal that shows when switching network





## **Related issues**

Contributes to solve issue: https://github.com/MetaMask/MetaMask-planning/issues/2796

## **Manual testing steps**

I have added screenshot below to see how it looks.

1. Go to the in-app brower on the mobile wallet, connect to a dapp like uniswap
2. Switch netowrk to a network for which permission was not added before
3. A modal will show up and show have the same margin width on the right side as on the left side.

## **Screenshots/Recordings**


| Before      | After    |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/de9491f2-5d61-4dc0-91bd-613cf8041124"> |<img width="350" alt="Screenshot 2024-04-18s at 3 56 43 PM" src="https://github.com/user-attachments/assets/c30a657d-62bf-4067-b11b-a4e0df24f74c"> |






## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
